CMakeLists.txt

```diff
@@ -210,10 +250,12 @@ target_link_libraries(
   kvsWebrtcClient
   PRIVATE kvspicUtils
           kvspicState
+          ${CMAKE_THREAD_LIBS_INIT}
```

Comment:
This is needed because mbedTLS doesn't link to pthread. If I didn't do this, the compilation will fail due to not able finding pthread APIs.

Reply:
What about other platforms like Windows

Reply:
@MushMal and I talked offline about this, and decided that we'll add Windows support later, not a part of this PR.